components/global/user_guide_dropdown/user_guide_dropdown.tsx

```diff
@@ -0,0 +1,130 @@
+
+class UserGuideDropdown extends React.PureComponent<Props, State> {
+    constructor(props: Props) {
+        super(props);
+        this.state = {
+            buttonActive: false,
+        };
+    }
+
+    toggleShortcutsModal = (e: MouseEvent) => {
+        e.preventDefault();
+        toggleShortcutsModal();
+    }
+
+    buttonToggleState = (menuActive: boolean) => {
+        this.setState({
+            buttonActive: menuActive,
+        });
+    }
+
+    askTheCommunityClick = () => {
+        trackEvent('ui', 'help_ask_the_community');
+    }
+
+    renderDropdownItems = (): React.ReactNode => {
+        const {intl} = this.props;
+
+        return (
+            <Menu.Group>
+                {this.props.enableAskCommunityLink === 'true' && (
+                    <Menu.ItemExternalLink
+                        id='askTheCommunityLink'
+                        url={askTheCommunityUrl}
+                        text={intl.formatMessage({id: 'userGuideHelp.askTheCommunity', defaultMessage: 'Ask the community'})}
+                        onClick={this.askTheCommunityClick}
+                    />
+                )}
+                <Menu.ItemExternalLink
+                    id='helpResourcesLink'
+                    url={this.props.helpLink}
+                    text={intl.formatMessage({id: 'userGuideHelp.helpResources', defaultMessage: 'Help resources'})}
+                />
+                <Menu.ItemExternalLink
+                    id='reportAProblemLink'
+                    url={this.props.reportAProblemLink}
+                    text={intl.formatMessage({id: 'userGuideHelp.reportAProblem', defaultMessage: 'Report a problem'})}
+                />
+                <Menu.ItemAction
+                    id='keyboardShortcuts'
+                    onClick={this.toggleShortcutsModal}
+                    text={intl.formatMessage({id: 'userGuideHelp.keyboardShortcuts', defaultMessage: 'Keyboard shortcuts'})}
+                />
+            </Menu.Group>
+        );
+    }
+
+    render() {
+        const {intl} = this.props;
+        const tooltip = (
+            <Tooltip
+                id='userGuideHelpTooltip'
+                className='hidden-xs'
+            >
+                <FormattedMessage
+                    id={'channel_header.userHelpGuide'}
+                    defaultMessage='Help'
+                />
+            </Tooltip>
+        );
+
+        return (
+            <MenuWrapper
+                className='userGuideHelp'
+                onToggle={this.buttonToggleState}
+            >
+                <OverlayTrigger
+                    delayShow={500}
+                    placement='bottom'
+                    overlay={this.state.buttonActive ? <></> : tooltip}
+                >
+                    <IconButton
+                        size={'sm'}
+                        icon={'help-circle-outline'}
+                        onClick={() => {}} // icon button currently requires onclick ... needs to revisit
```

Comment:
This seems like a design flaw in the icon button component.

Reply:
For sure. We will discuss aligning compass components with the needs for webapp in our team meeting today